GVRf/Framework/src/org/gearvrf/GVRInputManager.java

```diff
@@ -0,0 +1,242 @@
+package org.gearvrf;
+
+import java.io.IOException;
+import org.gearvrf.InputEvent.InputType;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.Message;
+import android.renderscript.Matrix4f;
+
+/**
+ * The {@link GVRInputManager} class handles all the input coming into GVRf
+ * using the call {@link #dispatchInputEvent(InputEvent)}
+ * 
+ * {@link InputEvent}s are dispatched across the attached scene graph to
+ * activate the related sensors in the graph.
+ * 
+ * For now we consider only one pointer (2D head tracked pointer). <br>
+ * 
+ */
+public class GVRInputManager {
+    private static final String TAG = GVRInputManager.class.getSimpleName();
+    private PointerSceneObject pointerSceneObject;
+    private final SensorManager sensorManager;
+
+    private ListenerDelegate listenerDelegate;
+
+    public interface InputEventListener {
+        public void onInputEvent(float[] values);
```

Comment:
Will add javadocs for this interface ... and probably rename it to a more meaningful name.